.murdock

```diff
@@ -329,24 +329,12 @@ run_test() {
 
 # execute static tests
 static_tests() {
-    local repo=${CI_BASE_REPO:-https://github.com/RIOT-OS/RIOT}
-    local branch=${CI_BASE_BRANCH:-master}
-
-    print_worker
```

Comment:
This is no longer included in the new output, is that intentional?

Reply:
nope, thanks! added it back